lua_examples/lfs/_init.lua

```diff
@@ -8,10 +8,10 @@
   module related initialisaion in this. This example uses standard Lua features to
   simplify the LFS API.
 
-  The first section adds a 'LFS' table to _G and uses the __index metamethod to
-  resolve functions in the LFS, so you can execute the main function of module
-  'fred' by executing LFS.fred(params), etc. It also implements some standard
-  readonly properties:
+  For Lua 5.1, the first section adds a 'LFS' table to _G and uses the __index 
+  metamethod to resolve functions in the LFS, so you can execute the main 
+  function of module 'fred' by executing LFS.fred(params), etc.
```

Comment:
+"For Lua 5.3, equivalent functionality is provided natively by the firmware" ?

Reply:
"For Lua 5.3 LFS table is populated by the LFS implementation in C so this part of the code is skipped." ?